src/test/java/com/epam/api/RunCucumberParallelTest.java

```diff
@@ -0,0 +1,14 @@
+package com.epam.api;
+
+import org.junit.platform.suite.api.SelectClasspathResource;
+import org.junit.platform.suite.api.Suite;
+import org.testng.annotations.DataProvider;
+
+@Suite
+@SelectClasspathResource("features/UserFiltersCreation.feature")
+public class RunCucumberParallelTest {
+    @DataProvider(parallel = true)
```

Comment:
Why TestNG data provider? 
Specified runner is JUnit - this will not work properly

Reply:
updated